plutus-core/plutus-core/src/PlutusCore/Parser.hs

```diff
@@ -101,18 +101,3 @@ parseTerm = parseGen term
 parseType :: (AsParserErrorBundle e, MonadError e m, MonadQuote m) =>
     Text -> m (Type TyName DefaultUni SourcePos)
 parseType = parseGen pType
-
-
-
-
-
--- >>> doIt term "(con bool True)"
--- Right (Constant (SourcePos {sourceName = "test", sourceLine = Pos 1, sourceColumn = Pos 2}) (Some (ValueOf bool True)))
--- >>> doIt term "(con (list bool) [True, False])"
--- Right (Constant (SourcePos {sourceName = "test", sourceLine = Pos 1, sourceColumn = Pos 2}) (Some (ValueOf list (bool) [True,False])))
--- >>> doIt term "(con (pair unit (list integer)) ((),[1,2,3]))"
--- Right (Constant (SourcePos {sourceName = "test", sourceLine = Pos 1, sourceColumn = Pos 2}) (Some (ValueOf pair (unit) (list (integer)) ((),[1,2,3]))))
--- >>> doIt term "(con (list (pair string bool)) [(\"abc\", True), (\"def\", False)])"
--- Right (Constant (SourcePos {sourceName = "test", sourceLine = Pos 1, sourceColumn = Pos 2}) (Some (ValueOf list (pair (string) (bool)) [("abc",True),("def",False)])))
--- >>> doIt term "(con string \"abc\")"
--- Right (Constant (SourcePos {sourceName = "test", sourceLine = Pos 1, sourceColumn = Pos 2}) (Some (ValueOf string "abc")))
```

Comment:
Could you create unit tests out of these? They are helpful examples.

Reply:
@gtrepta FYI, I [fixed](https://github.com/input-output-hk/plutus/pull/4724) rendering of built-in types yesterday, so these old GHCi sessions are no longer correct, you'll need to regerenerate the output for each test. Also, don't use `Show` like I did, use `Pretty` instead.